app/views/layouts/_header.html.erb

```diff
@@ -1,12 +1,32 @@
 <ul class="title-area">
   <li class="name">
     <h1 class="logo"><%= link_to "DBB", root_path %></h1>
-  </li>
   <li class="toggle-topbar menu-icon"><a href="#"><span></span></a></li>
+
 </ul>
 
 <section class="top-bar-section">
+  <ul class="left">
+    <a
+      href="https://twitter.com/betterbourbon" class="twitter-follow-button" data-show-count="false">
+    </a>
+      <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');
+      </script>
```

Comment:
Usually this script tag should go at the very top or bottom of the page depending on when it needs to load or in a separate js file entirely and then use a `javascript_include_tag 'filename'` on the page or layout. Try moving it to one of those places and making sure everything still works.